src/aliceVision/multiview/essentialF10Solver_test.cpp

```diff
@@ -0,0 +1,135 @@
+// This file is part of the AliceVision project.
+// Copyright (c) 2018 AliceVision contributors.
+// This Source Code Form is subject to the terms of the Mozilla Public License,
+// v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#define BOOST_TEST_MODULE F10Solver
+
+#include <aliceVision/multiview/essentialF10Solver.hpp>
+
+#include <boost/test/included/unit_test.hpp>
+#include <boost/test/floating_point_comparison.hpp>
+
+#include <vector>
+
+
+using namespace aliceVision;
+
+bool isEqual(const Mat first, const Mat second)
```

Comment:
can be replaced by `first.isApprox(second)`
eventually with a second argument that is the threshold 
https://eigen.tuxfamily.org/dox/classEigen_1_1DenseBase.html#ae8443357b808cd393be1b51974213f9c